### PR TITLE
feat(ghost-button): Add ghost button

### PR DESCRIPTION
**Pull request by tw15egan**
### Added
`html` and `scss` for `ghost-button`

### Changed
Matched focus style border changes to initial `border-width`

Closes https://github.ibm.com/Bluemix/carbon-issues/issues/137

### CODE REVIEW

**Comment by hellobrian** [comment]
👍 ✅

**Review by hellobrian — APPROVED** [review]
👍 ✅

**Comment by marijohannessen** [comment]
Super tiny thing, but the spacing between the text and the icon looks a bit too big:
screen shot 2017-05-26 at 10 59 15 am

Changing `margin-left` to `.275rem` seems to fix it:
screen shot 2017-05-26 at 11 00 39 am

**Comment by marijohannessen** [comment]
👍 ✅ ❗️

File: src/components/button/ghost-button.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<button class="bx--btn bx--btn--ghost" type="button">
+  Ghost
+  <svg class="bx--btn__icon">
+    <use xlink:href="/carbon-icons/bluemix-icons.svg#icon--add--glyph"></use>
```
Comment (hellobrian) [inline src/components/button/ghost-button.html]: These should be inline SVG

Comment (hellobrian) [reply src/components/button/ghost-button.html]:
```html
<svg class="bx--btn__icon" width="16" height="16" viewBox="0 0 16 16" fill-rule="evenodd">
  <path d="M8 0C3.6 0 0 3.6 0 8s3.6 8 8 8 8-3.6 8-8-3.6-8-8-8zm4 9H9v3H7V9H4V7h3V4h2v3h3v2z"></path>
</svg>
```

**Comment by hellobrian** [comment]
Not sure if you implemented all the changes you wanted just yet so I'm going to leave the merging up to you @tw15egan.

**Pull request merged**